File: 2185. Counting Words With a Given Prefix/2185. Counting Words With a Given Prefix.js
/**
 * @param {string[]} words
 * @param {string} pref
 * @return {number}
 */
var prefixCount = function(words, pref) {
  let res = 0
  for(let word of words){
    if (word.startsWith(pref)){
        res++
    }
  }
  return res  
};

File: 2185. Counting Words With a Given Prefix/2185. Counting Words With a Given Prefix.py
from typing import List
class Solution:
    def prefixCount(self, words: List[str], pref: str) -> int:
        res=0
        for word in words:
            if word.startswith(pref):
                res+=1
        
        return res

sol = Solution()
print(sol.prefixCount(words = ["pay","attention","practice","attend"], pref = "at"))